task3/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(MathFunction("x"));
+        }
+
+
+        public static double MathFunction(string a)
```

Comment:
Этот метод должен **только вычислять значение** функции от заданного аргумента
```suggestion
        public static double MathFunction(double x)
```